src/RoboSaverVirtualModule.sol

```diff
@@ -231,18 +233,31 @@ contract RoboSaverVirtualModule {
     function checker() external view returns (bool adjustPoolNeeded, bytes memory execPayload) {
         if (_isExternalTxQueued()) return (false, bytes("External transaction in queue, wait for it to be executed"));
 
-        /// @notice checks if there is a transaction queued up in the delay module by the virtual module itself
+        /// @dev check if there is a transaction queued up in the delay module by the virtual module itself
         if (txQueueData.nonce != 0) {
             return (true, abi.encodeWithSelector(this.adjustPool.selector, PoolAction.EXEC_QUEUE_POOL_ACTION, 0));
         }
 
         uint256 balance = EURE.balanceOf(CARD);
+        if (balance == 0) {
```

Comment:
thinking a bit more on this. this will block completly the case where the card actually has 0 eur balance, but bpts to wd and top it up, no?

Reply:
covered the case of zero bpt in 590bad501422147eca3b386e56fec86b69fdb3aa, lmk your thoughts 

also aim to push another ci task to ensure that whatever we push is compliant w/ the `forge fmt` formatting

Reply:
yes you are right, having 0 eure balance shouldnt be a showstopper in the case of having bpts

another approach could be `if bpt.bal ==  0 or eure.bal == 0 then revert`, but current solution works just as well

Reply:
re linting:

<img width="862" alt="Screenshot 2024-06-07 at 12 00 32" src="https://github.com/onchainification/robosaver/assets/2835259/61f24d1f-0390-4ef7-98e1-570a43c17f81">

Reply:
thanks for fixing the ci 🤝